lib/kubot/server.rb

```diff
@@ -0,0 +1,115 @@
+require 'http'
+require 'json'
+
+module Kubot
+  class MyServer < SlackRubyBot::Server
```

Comment:
Well, let's abstract from "I/My" thing and rename MyServer to SlackServer. Actually, I have no idea why they called it like "Server" because it's more about handlng, I would call it "SlackHandler". But it's up to you.